python/dbsp-api-client/dbsp_api_client/api/pipeline/http_output.py

```diff
@@ -183,6 +237,15 @@ async def asyncio_detailed(
         pipeline_id (str):
         table_name (str):
         format_ (str):
+        query (Union[Unset, None, OutputQuery]): A query over an output stream.
+
+            We currently do not support ad hoc queries.  Instead the client can use
```

Comment:
repeating this 3 times is not very maintainable, how about just a "see also"?

Reply:
This is auto-generated.